### PR TITLE
Design calculator

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this milestone, I did the following:
- Created a class based component for calculator
- Added style to calculator component
- Deleted some Create React App Boilerplates

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```
Comment (e71az) [inline src/App.js:9]: - [ ] The App.js component should be class-based, please change it in order to comply with the project requirements.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <div className="light-grey">8</div>
+        <div className="light-grey">9</div>
+        <div className="orange">x</div>
+        <div className="light-grey">4</div>
+        <div className="light-grey">5</div>
+        <div className="light-grey">6</div>
+        <div className="orange">-</div>
+        <div className="light-grey">3</div>
+        <div className="light-grey">2</div>
+        <div className="light-grey">1</div>
```
Comment (e71az) [inline src/components/Calculator.js:23]: - [ ] These numbers are inverted in order, please fix them in order to showcase a calculator equal to the design provided.

image

**Review by e71az — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @wuyepabdul, I am Elias, your reviewer for this PR

Well done so far with your project!

Your project is installing and running correctly, your calc design matches the one expected, great job so far with it, keep it up!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please remove all the unused files created while using the create react app command (like favicon, logos, and any other file used by the react page initialization) to comply with the project requirements. 
- [ ] Please check the comments below this PR for extra details and requirements.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @wuyepabdul, 

Good job on implementing the suggested changes. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**